bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -866,7 +912,7 @@ private void checkVersion(ShellyDeviceProfile prf, ShellySettingsStatus status)
             if (bindingConfig.autoCoIoT && ((version.compare(prf.fwVersion, SHELLY_API_MIN_FWCOIOT)) >= 0)
                     || (prf.fwVersion.equalsIgnoreCase("production_test"))) {
                 if (!config.eventsCoIoT) {
-                    logger.debug("{}: {}", thingName, messages.get("versioncheck.autocoiot"));
+                    logger.info("{}: {}", thingName, messages.get("versioncheck.autocoiot"));
```

Comment:
What's the reason you changed this?

Reply:
changed to debug, just to make sure

Reply:
There are the first devices, which are EOL, so I want to make the user aware that the firmware version does not support all required features